frontend/src/properties/Group.tsx

```diff
@@ -109,3 +111,15 @@ export const Group = ({ editingContextId, formId, group, widgetSubscriptions, se
     </div>
   );
 };
+const isTree = (widget: Widget): widget is Tree => widget.__typename === 'TreeWidget';
+  } else if (isTree(widget)) {
+    propertySection = (
+      <TreePropertySection
+        editingContextId={editingContextId}
+        formId={formId}
+        widget={widget}
+        key={widget.id}
+        subscribers={subscribers}
+        setSelection={setSelection}
+      />
+    );
```

Comment:
There are a couple of rebasing issues, in this file at least, which prevent the code from compiling. I'll handle them